src/bin/nydus-image/main.rs

```diff
@@ -848,13 +849,6 @@ impl Command {
         event_tracer!("euid", "{}", geteuid());
         event_tracer!("egid", "{}", getegid());
 
-        // Validate output bootstrap file
-        if !blob_inline_meta {
```

Comment:
It's ok to disable validation for create subcommand, but we'd better do `nydus-image check` after `nydus-image merge` in the converter tool.